### ISSUE TITLE
Add elfutils dev packages for libdw (and others)

### ISSUE THREAD

**Issue by mickours** [NixOS/nixpkgs#31940]
## Issue description
I'd like to have libdw dev package to build an other tool but I find out that it is not available.

There is a elfutils package already (``nixpkgs/pkgs/development/tools/misc/elfutils/default.nix``) but it do not provides .h files.

It would be interesting to provide several dev packages for each libs provided by elfutils, but I don't know how to do this. Any idea?

**Comment by jtojnar** [comment]
I can see the header file:

```shell
$ find $(nix-build -A elfutils) -name libdw.h
/nix/store/bcndqr34jbc8kigxdk07afrg34kq2s99-elfutils-0.170/include/elfutils/libdw.h
```

**Comment by mickours** [comment]
Thanks @jtojnar,

Actually, I found out that my problem comes from the options in my package. 

I add several dependencies to add for a specific features but I used ``optionnal`` instead of ``optionnals`` with an **s**. 

Like to add to my buildInputs I used ``++ stdenv.lib.optional myFeatureSupport [ libunwind elfutils]`` and it silently don't add it my list of inputs. :-/  

Thanks again and sorry for that.
